ui/ui-components/components/record/SampleRecordCard.tsx

```diff
@@ -366,6 +366,176 @@ const SampleRecordCard: React.FC<SampleRecordCardProps> = ({
     setExporting(false);
   }, [client, destinationId, record?.id]);
 
+  const content = useMemo(() => {
```

Comment:
Hard to compare with the one below because of the move. Did anything change in particular or just move up/memoizing?

Reply:
Added a comment around the new bit.